package.json

```diff
@@ -23,6 +23,7 @@
     "babel-core": "^6.26.0",
     "babel-jest": "^21.0.2",
     "babel-loader": "^7.1.2",
+    "babel-plugin-transform-class-properties": "^6.24.1",
```

Comment:
Just curious, where is this needed?

Reply:
I was going to use it and ended up not. Nice to have, but I can remove it.